packages/dev/core/src/Gizmos/rotationGizmo.ts

```diff
@@ -215,6 +215,16 @@ export class RotationGizmo extends Gizmo implements IRotationGizmo {
         return this.xGizmo.updateGizmoRotationToMatchAttachedMesh;
     }
 
+    public set updateGizmoPositionToMatchAttachedMesh(value: boolean) {
+        if (this.xGizmo) {

```

Comment:
It seems you don't need this check as `xGizmo/yGizmo/zGizmo` are created in the constructor (but I understand you simply copied `updateGizmoRotationToMatchAttachedMesh` which is doing the check).